src/test/scala/org/broadinstitute/dsde/firecloud/utils/StreamingPassthroughSpec.scala

```diff
@@ -57,38 +57,38 @@ class StreamingPassthroughSpec extends AnyFreeSpec
   }
 
 
-  "convertToTargetUri" - {
+  "convertToRemoteUri" - {
     "should calculate a remainder" in {
       val requestUri = Uri("http://localhost:8123/foo/bar/baz/qux")
       val localBasePath = Path("/foo/bar")
       val remoteBaseUri = Uri("https://example.com/api/version/foo")
 
       val expected = Uri("https://example.com/api/version/foo/baz/qux")
-      convertToRemoteUri(requestUri, localBasePath, remoteBaseUri) shouldBe expected
+      convertToRemoteUri(requestUri, localBasePath, remoteBaseUri, None) shouldBe expected
```

Comment:
I think the `None` inputs can be removed in this file? Unless you're deliberately making clear what the test input is.

Reply:
Yeah, I usually write out the parameters on tests just so the function outline/context is clear